addon/templates/components/frost-list-content-container.hbs

```diff
@@ -1,44 +1,31 @@
 {{! Template for the frost-list-content-container component }}
+{{#vertical-collection
+  alwaysRemeasure=true
+  alwaysUseDefaultHeight=alwaysUseDefaultHeight
+  bufferSize=bufferSize
+  content=items
+  defaultHeight=defaultHeight
+  firstReached=onLoadPrevious
+  lastReached=onLoadNext
+  scrollPosition=scrollTop
+  useContentProxy=false
 
-{{#if pagination}}
```

Comment:
I can remove it. I thought you asked me to replace the `each` by a vertical-collection.